challenge_set_the_table/src/challenge_set_the_table/navigate_to_and_place_item_on_table.py

```diff
@@ -253,23 +248,25 @@ def _publish_item_poses(robot, items):
 if __name__ == '__main__':
     rospy.init_node(os.path.splitext("test_" + os.path.basename(__file__))[0])
 
-    item_frames = {k:item_vector_to_item_frame(v) for k, v in item_vector_dict.items()}
+    item_frames = {k: item_vector_to_item_frame(v) for k, v in item_vector_dict.items()}
     import pprint
     print('Item frames:')
     pprint.pprint(item_frames)
 
-    item_poses = {k:item_frame_to_pose(v, 'kitchen_table') for k, v in item_frames.items()}
+    item_poses = {k: item_frame_to_pose(v, 'kitchen_table') for k, v in item_frames.items()}
     print('Item poses:')
     pprint.pprint(item_poses)
 
-    hero = Hero()
-    _publish_item_poses(hero, item_poses)
-    hero.reset()
+    robot_instance = get_robot("hero")
+    _publish_item_poses(robot_instance, item_poses)
+    robot_instance.reset()
     try:
-        placement_height = float(sys.argv[2])
+        placement_height_ = float(sys.argv[2])
```

Comment:
Any reason for the trailing underscore? I know in C++ is is common to do with private class variables, but that isn't the case here.

Reply:
Same reason as before, only to differentiate the variable name from the 'placement_height' that is used in the state.